src/utils/get-comment-registry.ts

```diff
@@ -243,7 +244,7 @@ export const getCommentRegistryFromImportDeclarations = ({
     /** Constructed Output Nodes */
     outputNodes: ImportDeclaration[];
 }) => {
-    if (outputNodes.length === 0 || !firstImport) {
+    if (outputNodes?.length === 0 || !firstImport) {
```

Comment:
When would `outputNodes` be undefined?  We have TS to guarantee it's an array just above, right?

Reply:
I concur that TypeScript would prevent that. 

I wasn’t sure how robust we wanted our `src/utils/*` to be against misuse.

I wrote a unit test that crashed this, so I made the check verify that it was non-null. All of my `Fatal Internal Error` checks could be removed if you wanted to do so. I found them helpful during the initial implementation phase.